symphony/lib/toolkit/class.widget.php

```diff
@@ -403,7 +403,7 @@ public static function TableHead(array $columns = null)
                     $th->setAttribute('scope', $col[1]);
                 }
 
-                if (is_array($col[2]) && !empty($col[2])) {
+                if (!empty($col[2]) && is_array($col[2])) {
```

Comment:
Should we create method in the General for those uses cases ? `isset + empty` could also be needed.